### ISSUE TITLE
keep version of greenwood/cli version output as part of init in sync with latest stable CLI version

### ISSUE THREAD

**Issue by thescientist13** [ProjectEvergreen/greenwood#781]
## Type of Change
- [ ] New Feature Request
- [ ] Documentation / Website
- [x] Improvement / Suggestion
- [ ] Bug
- [ ] Other (please clarify below)

## Summary
Currently as part of #778 and related to #770 , we are using a hardcoded _package.json_ to use as the template for scaffolding.   This means the version of the [**@greenwood/cli** in the generated _package.json_](https://github.com/ProjectEvergreen/greenwood/tree/master/packages/init/src/template/package.json) will never bump unless we change it ourselves.  It would be good to find a mechanism to try and keep this up to date as part of each new release.

## Details
Basically, the init package should pull from its own _package.json_ each time and use that as the installed version.  The only question is, if it's an `alpha.x` release though, e.g. `^0.19.0-alpha.1` we should probably fallback to last minor, e.g. `^0.18.0`.

**Comment by thescientist13** [comment]
One tack on thing is that I had [turned off `stdio`](https://github.com/ProjectEvergreen/greenwood/pull/778/files#diff-276757543950e6c5c92261a791b38ace541c72d56a907556529f742066dd778dR101) because warning seemed to make Yarn fail the test.  Might be good to revaluate since having the `install` logs in the terminal output for the user would be nice.
